Source/WebCore/css/CSSPrimitiveValue.cpp

```diff
@@ -688,8 +688,16 @@ double CSSPrimitiveValue::computeLengthDouble(const CSSToLengthConversionData& c
         // The multiplier and factor is applied to each value in the calc expression individually
         return m_value.calc->computeLengthPx(conversionData);
     }
+    auto type = primitiveType();
 
-    return computeNonCalcLengthDouble(conversionData, primitiveType(), m_value.num);
+    // FIXME: Without this, imported/w3c/web-platform-tests/css/css-properties-values-api/unit-cycles.html would
```

Comment:
@anttijk & @litherum I could really use your help here understanding what I am doing wrong. I have no idea why FontCascade::m_fonts is uninitialized here and how to fix it.

Note that that test in question was relying on StylePropertyMap::set() to actually run so there is also a small chance this exposed a pre-existing bug.

Reply:
I think you can just skip the test. The crash involves the unrelated and less-than-complete registerProperty feature that is not a shipping configuration. I plan to work on it later.